cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -101,10 +199,9 @@ protected String getInstanceURI() {
 
   /**
    * CDAP access token for making requests to secure CDAP instances.
-   * Can be obtained via the CDAP authentication server.
    */
-  protected String getAccessToken() {
-    return System.getProperty("accessToken", "");
+  private AccessToken getAccessToken() {
```

Comment:
i am not sure if we want to allow that though, in most cases the derived classes would do ``getConnectionConfig`` and that will set accessToken.